tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -163,10 +174,86 @@
                        subjects_dir=subjects_dir, dig=True,
                        surfaces=['head-dense', 'white'], coord_frame='meg')
 
+
 ###############################################################################
 # It is quite clear that the MRI surfaces (head, brain) are not well aligned
 # to the head digitization points (dots).
 #
+# Compute the transform
+# ---------------------
+# Let's step through the process that the transform is accomplishing.
+#
+# First, let's define a helper function to plot the alignment of the
+# points with the head using voxel coordinates.
+
+def plot_dig_alignment(points, coord_system='ras'):
+    renderer_kwargs = dict(bgcolor='w', smooth_shading=False)
+    renderer = mne.viz.backends.renderer._get_renderer(
+        size=(800, 400), **renderer_kwargs)
+    seghead_rr, seghead_tri = mne.read_surface(os.path.join(
+        subjects_dir, 'sample', 'surf', 'lh.seghead'))
+    renderer.mesh(*seghead_rr.T, triangles=seghead_tri, color=(0.7,) * 3,
+                  opacity=0.2)
+    for point in points.copy():
+        if coord_system == 'vox':
+            point -= 128  # RAS zero centered
+            center = (point[0], point[2], -point[1])
+        else:
+            center = point
+        renderer.sphere(center=center, color='r', scale=5)
+    view_kwargs = dict(elevation=90, azimuth=0)
+    view_kwargs['focalpoint'] = (0., 0., 0.)
+    mne.viz.set_3d_view(figure=renderer.figure, distance=1000, **view_kwargs)
+    renderer.show()
+
+
+###############################################################################
+# Get landmarks in head space from the DigMontage stored in raw
+# -------------------------------------------------------------
+# Adjust units by * 1e3 m -> mm.
+#
+# The plot below shows the digitization points in the coordinate space
+# native to the digitizer equipment plotted with the head surface in
+# right-anterior-superior (RAS) coordinates. The relationship between the
+# 3D points in the head space would fit on the head but the transformation from
+# the coregistration has not been applied yet so there is a mismatch.
+
+head_space = np.array([dig['r'] for dig in
+                       raw.info['dig']], dtype=float) * 1e3
+plot_dig_alignment(head_space)
+
+###############################################################################
+# Transform digitization points into MRI space
+# --------------------------------------------
+# Rotate and translate the points based on the coregistration.
+#
+# The plot below shows the head space coordinates transformed to mri
+# RAS coordinates. It correctly aligns to the head, which is also in RAS,
+# but still has to be transformed to the voxel coordinate space.
+
+trans_mm = trans.copy()
+trans_mm['trans'][:3, 3] *= 1e3
```

Comment:
why moving to mm ? if you scale to mm in the plot_dig_alignment then maybe you can simplify this?
maybe what is easier is to display the freesurfer surface in meters ?